core/bin/zksync_api/src/api_server/rest/v01/api_impl.rs

```diff
@@ -65,27 +66,61 @@ impl ApiV01 {
                 .expect("TickerConfig::liquidity_volume must be positive"),
         );
 
+        let ticker_config = zksync_config::configs::TickerConfig::from_env();
+        let unconditionally_valid_addresses = ticker_config
```

Comment:
its not more easy just to get the vec, without mapping to string, since the Token object also have the address as a h160?

Reply:
hm true that